src/main/java/dk/sdu/se_f22/brandmodule/stemming/ExceptionUtilities.java

```diff
@@ -2,37 +2,63 @@
 
 import dk.sdu.se_f22.sharedlibrary.db.DBConnection;
 
+import javax.xml.transform.Result;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.util.HashMap;
 
 public class ExceptionUtilities {
     public static void createException(String name) throws SQLException {
-        String query = String.format("INSERT INTO StemmingException (name) VALUES ('%s')", name);
-        ResultSet result = DBConnection.getConnection().createStatement().executeQuery(query);
-        result.close();
+        Connection connection = DBConnection.getPooledConnection();
+        PreparedStatement statement = connection.prepareStatement("INSERT INTO stemmingexceptions (exception) VALUES (?)");
+        statement.setString(1, name);
+        statement.executeUpdate();
+        connection.close();
     }
+
     public static HashMap<Integer, String> getExceptions() throws SQLException {
-        String query = String.format("SELECT * FROM StemmingException");
-        ResultSet result = DBConnection.getConnection().createStatement().executeQuery(query);
-        result.close();
+        Connection connection = DBConnection.getPooledConnection();
+        String query = "SELECT * FROM stemmingexceptions";
+        ResultSet result = connection.prepareStatement(query).executeQuery();
         return handleResult(result);
-
     }
-    public static HashMap<Integer, String> findException(String name) throws SQLException {
-        String query = String.format("SELECT * FROM StemmingException WHERE name = '%s';", name);
-        ResultSet result = DBConnection.getConnection().createStatement().executeQuery(query);
-        result.close();
+
+    public static HashMap<Integer, String> getException(String name) throws SQLException {
+        Connection connection = DBConnection.getPooledConnection();
+        PreparedStatement statement = connection.prepareStatement("SELECT * FROM stemmingexceptions WHERE exception = ?");
+        statement.setString(1, name);
+        ResultSet result = statement.executeQuery();
         return handleResult(result);
+
+    }
+
+    public static void updateException(String name, String newName) throws SQLException {
+        Connection connection = DBConnection.getPooledConnection();
+        PreparedStatement statement = connection.prepareStatement("UPDATE stemmingexceptions SET exception = ?" + "WHERE exception = ?");
```

Comment:
Where is this prepared statement closed? 
A preparedstatement, once queried, opens a cursor in the Database.